components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1219,11 +1230,11 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
          call handle_error(clip_values( &
             tmid(1:ncol,1:nlev_rad), k_dist_lw%get_temp_min(), k_dist_lw%get_temp_max(), &
             trim(subname) // ' tmid' &
-         ), fatal=.false.)
+         ), fatal=.false., warn=rrtmgp_enable_limiter_warnings)
```

Comment:
Did you mean to only apply this to temperature limiters? If so, I'd suggest renaming the namelist variable to something like "rrtmgp_enable_temperature_warnings", since we now have limiters for optical properties as well. If you meant to apply it to all limiters, then you should add this clause to the other calls to `clip_values`. I'd prefer the former, and then if we want to disable warnings about optical properties we can control that separately (at a later time maybe).

Reply:
I was initially going to apply to other limiters, but I realized later that the temperature ones were the only problematic ones. I'll change the name.